nano/node/vote_processor.hpp

```diff
@@ -33,7 +33,7 @@ namespace transport
 class vote_processor final
 {
 public:
-	explicit vote_processor (nano::signature_checker & checker_a, nano::active_transactions & active_a, nano::node_observers & observers_a, nano::stat & stats_a, nano::node_config & config_a, nano::node_flags & flags_a, nano::logger_mt & logger_a, nano::online_reps & online_reps_a, nano::ledger & ledger_a, nano::network_params & network_params_a);
+	explicit vote_processor (nano::node & node_a);
```

Comment:
Many class forward declarations can be removed in this file, and `node` should be declared. Does this change to reference `node` go against our efforts to not have `node` be referenced by other classes to reduce compilation time?

Reply:
I think this was some cruft left over from building the PR. At any rate it's an unrelated change so removed.